lib/ext/provider.rb

```diff
@@ -19,6 +19,7 @@ def self.generate_provider(options = {})
     prov.family_name = NAMES_RANDOM['last'].sample
     prov.npi = NpiGenerator.generate
     prov.tin = rand.to_s[2..10]
+    prov.ccn = rand(1..66).to_s.ljust(2, '0') + rand(1..9899).to_s
```

Comment:
could the left side of the addition be simplified to `rand(10..66).to_s` or is that different from what you want?